tutorials/algorithms/02_vqe_convergence.ipynb

```diff
@@ -271,10 +276,8 @@
    "outputs": [],
    "source": [
     "import logging\n",
-    "from qiskit.aqua import set_qiskit_aqua_logging\n",
-    "\n",
-    "\n",
-    "set_qiskit_aqua_logging(logging.INFO)"
+    "logging.basicConfig(level=logging.INFO)\n",
+    "logging.getLogger('qiskit.algorithms').setLevel(logging.INFO)"
```

Comment:
Since we could not really run and show logging I had just copy/paste a couple of lines from an actual run into the notebook in the past. I think the lines are still valid as such but we should edit out the `.aqua` in them which would make them correct for the current code.